docs/pages/build/android-builds.md

```diff
@@ -0,0 +1,125 @@
+
+## Build Process
+
+Let's take a closer look at the steps for building Android projects with EAS Builds.
+
+### Local Steps
+
+The first phase is happening on your computer, Expo CLI is in charge of completing the following steps:
+
+1. Check if the git index is clean - this means if there aren't any uncommitted changes. If it's not clean an error is thrown.
+2. Prepare the credentials needed for the build unless `builds.android.PROFILE_NAME.withoutCredentials` is set to `true`.
+
+   Depending on the value of `builds.android.PROFILE_NAME.credentialsSource`, the credentials are obtained from either the local `credentials.json` file or from the Expo servers. If the `auto` or `remote` mode is selected but no credentials exist yet, you're offered to generate a new keystore.
+
+3. Check if the Android project is configured so it could be built on the EAS Builds servers.
+
+   Namely, Expo CLI is checking whether `android/app/build.gradle` contains `apply from: "./eas-build.gradle"`.
+   If the project is not configured, run auto-configuration steps ([learn more below](#project-auto-configuration)).
+
+4. Create the tarball containing all your project sources - run `git archive --format=tar.gz --prefix project/ -o project.tar.gz HEAD`.
+5. Upload the project tarball to AWS S3 and send the build request to EAS Builds.
+
+### Remote Steps
+
+Next, this is what happens when EAS Builds picks up your request:
+
+1. Create a new Docker container for the build.
+
+   Every build gets its own, fresh container with all build tools installed there (Java JDK, Android SDK, NDK, and so on).
+
+2. Download the project tarball from AWS S3 and unpack it.
+3. Run `yarn install` in the project root.
+4. Restore the keystore (if it was included in the build request).
+5. Run `./gradlew COMMAND` in the `android` directory inside your project.
+
+   `COMMAND` is the command defined in your `eas.json` at `builds.android.PROFILE_NAME.gradleCommand`. It defaults to `:app:bundleRelease` which produces the AAB (Android App Bundle).
+
+6. Upload the build artifact to AWS S3.
+
+   The artifact path can be configured in `eas.json` at `builds.android.PROFILE_NAME.artifactPath`. It defaults to `android/app/build/outputs/bundle/release/app-release.aab`.
+
+## Project Auto-Configuration
+
+Every time you want to build a new Android app binary, we're checking if the project is set up correctly so we can semlessly run the build process on our servers.
+
+### Android Keystore
+
+Android requires you to sign your application with a certificate. That certificate is stored in your keystore. The Google Play store identifies applications based on the certificate. It means that if you lose your keystore you may not be able to update your application in the store. However, with [Play App Signing](https://developer.android.com/studio/publish/app-signing#app-signing-google-play), you can mitigate the risk of losing your keystore.
```

Comment:
```suggestion
Android requires you to sign your application with a certificate. That certificate is stored in your Keystore. The Google Play store identifies applications based on the certificate. It means that if you lose your Keystore you may not be able to update your application in the store. However, with [app signing by Google Play](https://developer.android.com/studio/publish/app-signing#app-signing-google-play), you can mitigate the risk of losing your keystore.
```

Reply:
`app signing by Google Play` is used in google documentation

Reply:
No, it's not. Open the link :)